.github/workflows/generate-chart-readme.yaml

```diff
@@ -0,0 +1,37 @@
+name: 'Hlem Readme Generator'
+
+on:
+  pull_request_target:
```

Comment:
Since this runs on `pull_request_target`, this will need to be in the base before it will push changes on PRs.